src/engine/specification.md

```diff
@@ -153,6 +154,8 @@ This structure contains the attributes required to initiate a payload build proc
 
 4. If any of the above fails due to errors unrelated to the client software's normal `SYNCING` status, the client software **MUST** return an error.
 
+5. Client software **MAY** use disparities in `headBlockHash` and `safeBlockHash` in decisions about what information to surface in user APIs.
```

Comment:
I'd defer this statement to JSON-RPC API spec change

Reply:
yeah, that's fine

Reply:
done